locale/circuitpython.pot

```diff
@@ -1336,10 +1332,6 @@ msgstr ""
 msgid "Pull not used when direction is output."
 msgstr ""
 
-#: ports/stm/ref/pulseout-pre-timeralloc.c
-msgid "PulseOut not supported on this chip"
```

Comment:
Why is this being deleted? Maybe you need to merge adafruit/main into your branch?

Reply:
Nevermind. This was my mistake in my PR. This was an error I added but deleted before merging. :roll_eyes: